### ISSUE TITLE
Truffle develop showing no logs

### ISSUE THREAD

**Issue by boorac** [trufflesuite/truffle#846]
- [ *] I've asked for help in the [Truffle Gitter](http://gitter.im/Consensys/truffle) before filing this issue.

---------------------------

## Issue

Commands:
```
truffle develop
truffle develop --log
```

show no output on console

## Steps to Reproduce

As of `Truffle v4.1.3 (core: 4.1.3)` problem started from day 1.

## Expected Behavior

I should see console output (e.g. accounts etc)

## Actual Results
No logs are shown

## Environment

* Operating System: `Ubuntu 16.04`
* Ethereum client: `geth Version: 1.8.2-stable`
* Truffle version: `Truffle v4.1.3 (core: 4.1.3), Solidity v0.4.19 (solc-js)`:
* node version: `v8.10.0`
* npm version: `5.6.0`

**Comment by boorac** [comment]
Reinstalling node and truffle seems to fix the issue.

**Comment by kiview** [comment]
Experiencing a similar behavior with
```
Truffle v5.2.3 (core: 5.2.3)
Solidity v0.5.16 (solc-js)
Node v14.16.0
Web3.js v1.2.9
```
on WSL2 on Ubuntu 20.04.

Running `truffle develop --log` will terminate the process after initial output.

**Comment by fkirc** [comment]
> Experiencing a similar behavior with
> 
> ```
> Truffle v5.2.3 (core: 5.2.3)
> Solidity v0.5.16 (solc-js)
> Node v14.16.0
> Web3.js v1.2.9
> ```
> 
> on WSL2 on Ubuntu 20.04.
> 
> Running `truffle develop --log` will terminate the process after initial output.

I am also running into the same issue with the exact same versions.
Firstly, I run `truffle develop` (worked as expected).
However, I got the following in a second terminal window:

`$ truffle develop --log`
`Connected to existing Truffle Develop session at http://127.0.0.1:8545/`
-> Immediate termination

This output tells me that it connected to my existing Truffle Develop session, but then the command terminates immediately, which does not make any sense for me.
Instead, I expected `truffle develop --log` to remain open in order to see ongoing transactions (as described in https://github.com/trufflesuite/truffle/issues/657).

I also tried to first run `truffle develop --log` before running `truffle develop`, but that did not work either.
